lib/util/exec/docker/index.ts

```diff
@@ -208,7 +208,7 @@ export async function removeDanglingContainers(): Promise<void> {
 
 export async function generateDockerCommand(
   commands: string[],
-  preCommands: string[],
+  preCommands: Opt<string>[],
```

Comment:
nope, caller needs to filter with `is.string` instead of `Boolean` to get right type

Reply:
`preCommands` is used here and has `Opt<string>[]` type so I thought it was okay to update the type in `generateDockerCommand` .